frontend/src/concepts/modelRegistry/const.ts

```diff
@@ -1,2 +1,2 @@
-export const MODEL_REGISTRY_DEFINITION_NAME = 'modelregistry-sample'; // TODO: harcoded value, we need to dynamically fetch this
-export const MODEL_REGISTRY_DEFAULT_NAMESPACE = 'shared'; // TODO: harcoded value, we need to check which will be the default namespace
+export const MODEL_REGISTRY_DEFAULT_NAMESPACE = 'odh-model-registries';
```

Comment:
Will this be managed for RHOAI somehow? I assume we need a TODO here

Reply:
Yes, this is managed by RHOAI, I'm sorry there's a TODO but it was mixed when i split up the PR, let me add it again here